include/merlin/debug.hpp

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
Maybe `debug_utils.hpp` would be better, for avoiding some name conflict, debug is too common name.